### DISCUSSION TITLE
Elastalert issue when using ARM instance and docker-compose

### DISCUSSION THREAD

**Discussion by Shubham1406220** [jertel/elastalert2#1088] [Q&A] [answered]
`docker-compose version 1.29.2`

Starting elastalert ... error

ERROR: for elastalert  Cannot start service elastalert: invalid task create option for io.containerd.runtime.v1.linux: unknown

ERROR: for elastalert  Cannot start service elastalert: invalid task create option for io.containerd.runtime.v1.linux: unknown
ERROR: Encountered errors while bringing up the project.

**Comment by jertel** [comment, marked as answer]
This looks like a container runtime problem. Are you able to launch other containers via docker compose? Perhaps check with the containerd team on what to look for with that error, as it appears to be an internal error from what I can see in that source code.

https://github.com/containerd/containerd/blob/3f565daf682f652a13bbafeff63bc618377e1b71/services/tasks/local.go#L802
